google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java

```diff
@@ -423,10 +414,16 @@ private List<SessionImpl> internalBatchCreateSessions(
 
   /** Returns a {@link SessionImpl} that references the existing session with the given name. */
   SessionImpl sessionWithId(String name) {
+    return sessionWithId(name, /*isMultiplexedSession= */ false);
+  }
+
+  /** Returns a {@link SessionImpl} that references the existing session with the given name. */
+  SessionImpl sessionWithId(String name, boolean isMultiplexedSession) {
```

Comment:
We should not add a boolean that indicates whether the session is a multiplexed session or not here, as the caller does not know. The session might have been created by a  different client, and this client does not know whether the other client created a regular or a multiplexed session.